src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,16 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ postList, users, comments }) => (
```

Comment:
```suggestion
export const PostList = ({ posts }) => (
```
it should be an array of posts where user information and comments array will be as the fields in the post objects